enforcer-rules/src/it/bom-project-error/verify.groovy

```diff
@@ -1,8 +1,16 @@
-def buildLog = new File(basedir, "build.log")
+def buildLog = new File(basedir, "build.log").text.replaceAll("\\r", "")

```

Comment:
why do we need to do this?
what if it's only \r we're removing and there aren't any \n?
at the least this should convert \r\n to \n rather than stripping \r

Reply:
Updated to replace "\r\n".

Reply:
Now Windows build fails. Investigating.